sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -116,45 +121,48 @@ You will also need to [register a new AAD application][register_aad_app] and [gr
 
 ## Key concepts
 
-### FormRecognizerClient
+### DocumentAnalysisClient
 
-`FormRecognizerClient` provides operations for:
+`DocumentAnalysisClient` provides operations for analyzing input documents using custom and prebuilt models through the `StartAnalyzeDocument` and `StartAnalyzeDocumentFromUri` APIs. Use the `modelId` parameter to select the type of model for analysis.
 
-- Recognizing form fields and content, using custom models trained to recognize your custom forms.  These values are returned in a collection of `RecognizedForm` objects. See example [Recognize Custom Forms](#recognize-custom-forms).
-- Recognizing form content, including tables, lines, words, and selection marks like radio buttons and check boxes without the need to train a model.  Form content is returned in a collection of `FormPage` objects. See example [Recognize Content](#recognize-content).
-- Recognizing common fields from the following form types using prebuilt models. These fields and meta-data are returned in a collection of `RecognizedForm` objects.
-Supported prebuilt models:
-  - Receipts
-  - Business cards
-  - Invoices
-  - Identity Documents
+|Model ID|Features
+|-|-
+|"prebuilt-layout"| Text extraction, selection marks, tables
+|"prebuilt-document"| Text extraction, selection marks, tables, key-value pairs and entities
+|"prebuilt-invoices"| Text extraction, selection marks, tables, and pre-trained fields and values pertaining to invoices
+|"prebuilt-businessCard"| Text extraction and pre-trained fields and values pertaining to English business cards
+|"prebuilt-idDocument"| Text extraction and pre-trained fields and values pertaining to US driver licenses and international passports
+|"prebuilt-receipt"| Text extraction and pre-trained fields and values pertaining to English sales receipts
+|"{custom-model-id}"| Text extraction, selection marks, tables, labeled fields and values from your custom documents
+
+More information about analyzing documents, including supported features and locales can be found in the [service documentation][formreco_models].
 
-### FormTrainingClient
+### DocumentModelAdministrationClient
 
-`FormTrainingClient` provides operations for:
+`DocumentModelAdministrationClient` provides operations for:
 
-- Training custom models to recognize all fields and values found in your custom forms.  A `CustomFormModel` is returned indicating the form types the model will recognize, and the fields it will extract for each form type.
-- Training custom models to recognize specific fields and values you specify by labeling your custom forms.  A `CustomFormModel` is returned indicating the fields the model will extract, as well as the estimated accuracy for each field.
+- Building custom models to analyze specific fields you specify by labeling your custom documents. A `DocumentModel` is returned indicating the document type the model can analyze, the fields it can analyze for each document type, and the estimated accuracy for each field.
```

Comment:
Nit? 
```suggestion
- Building custom models to analyze specific fields you specify by labeling your custom documents. A `DocumentModel` is returned indicating the document type(s) the model can analyze, the fields it can analyze for each document type, and the estimated accuracy for each field.
```

Reply:
If not adding the `(s)` then maybe this:
```suggestion
- Building custom models to analyze specific fields you specify by labeling your custom documents. A `DocumentModel` is returned indicating the document type the model can analyze, the fields it can analyze for that document type, and the estimated accuracy for each field.
```